### DISCUSSION TITLE
Quality issues video playback

### DISCUSSION THREAD

**Discussion by manuelveith** [mediacms-io/mediacms#523] [Q&A]
Hi,

we post training videos on our internal platform and I get some troubles with image quality when playing them back through video.js. 

First I though it was a problem with encoding profiles, but I copied the transcoded files (1080p-H264 only) over to my PC to check it directy in VLC.
Then I played the m8u playlist directly in the Edge browser and the image was as sharp as the original.
Playing the video in MediaCMS, it's slightly blurred. It does not matter if I have it fullscreen or not. Best seen on text in the video.

I checked in dev mode if it plays the correct files, also renamed the HLS folder to enforce playback of the transcoded mp4 directly.

So far I know that the browser can play it back correct, so I assume it must be something with the video.js player. I could not find any option in video.js that could have impact on rendered outcome.

Does anynone has similar problems?

System is  Windows 10, NVidia P2000 card on a 4k monitor.

**Comment by mgogoulos** [comment]
Hi, I cannot reproduce this, If you want either post a video on https://demo.mediacms.io/ or on your portal (assuming it has a public URL) and link it here so we can try it and see how it plays and if we can reproduce what you describe

**Comment by manuelveith** [comment]
Hi,

I made a short clip and uploaded it on the demo server https://demo.mediacms.io/view?m=CtHJPaLbE

You are right, it look fine when viewed on a 1080p monitor. But when I view it on a 4k monitor in a window, the fonts are blurry, like a 720p video on a 1080p screen. I guess thats a viewer scaling problem. Is there any option to set a fixed size or disable rescaling when the window is riszed?
Using the HTML5 default player in the browser, it does not impact the quality when resizing the window.
